docs/Developers/Development_Setup.rst

```diff
@@ -91,30 +91,24 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the master branch for latest updates
+ and its dependencies with pip3 using
 
 ::
 
     (coala-venv)$ git clone https://github.com/coala/coala
     (coala-venv)$ cd coala
     (coala-venv)$ pip3 install .
+    # This will install coala from the current directory.
     (coala-venv)$ cd -
     (coala-venv)$ git clone https://github.com/coala/coala-bears
     (coala-venv)$ cd coala-bears
     (coala-venv)$ pip3 install .
+    # This will install coala-bears from the current directory.
 
-Once you have forked the repository you would like to modify, you can
-install a repository-backed version of the repository using
-
-::
-
-    (coala-venv)$ pip3 install -e <path/to/forked/repository>
-
-You will then be able to edit the repository and have the changes take effect
-in your virtualenv immediately. You will also be able to use pip3 to manage
-your installation of the package should you need to install from a different
-source in the future.
+Once you have forked the repository, you will be able to edit the repository.
+pip3 will help in managing the installed package so that you can easily remove
+the package, if you feel the need of installing from a different source.
```

Comment:
>Once you have forked the repository, you will be able to edit the repository.

What does this additions mean? Isn't it all obvious. In the issue its asked to remove just the second code sample, not the description. The description should be added to the description of the first code sample.

Also this line shouldn't be removed at all
>You will then be able to edit the repository and have the changes take effect in your virtualenv immediately.